osx/make-installer.sh

```diff
@@ -120,10 +120,10 @@ install_name_tool -change "${HOMEBREW_PREFIX}/opt/pcre2/lib/libpcre2-8.0.dylib"
 cp -r "${HOME}/.luarocks/lib/lua/5.1/luasql" "${app}/Contents/MacOS"
 cp -v ${HOMEBREW_PREFIX}/opt/sqlite/lib/libsqlite3.0.dylib  "${app}/Contents/Frameworks/"
 install_name_tool -id  "@executable_path/../Frameworks/libsqlite3.0.dylib" "${app}/Contents/Frameworks/libsqlite3.0.dylib"
-# need to adjust sqlite3.so manually as it is a level lower than expected...
```

Comment:
Is that because the `luasql-sqlite3.so` file produced from the `luasqu-sqlite3` luarock is placed in a `Luasql` sub-directory (in the same manner as the `zip.so` one from the `lua-zip` rock ends up in a `brimworks` sub-directory)?

This is from my current (Windoze) box but the same principle applies...
<img width="740" height="721" alt="image" src="https://github.com/user-attachments/assets/72e68e8b-8734-4d4a-92de-19fcb843169e" />

<img width="821" height="468" alt="image" src="https://github.com/user-attachments/assets/d23e9033-3b18-4325-8d03-3ceae6854d26" />